File: CSharp/armstrong.cs
using System;

class Program
{
    static void Main()
    {
        int num, sum = 0, digit, temp;

        Console.Write("Enter a number: ");
        num = Convert.ToInt32(Console.ReadLine());

        temp = num;

        while (temp != 0)
        {
            digit = temp % 10;
            sum += digit * digit * digit;
            temp /= 10;
        }

        if (sum == num)
            Console.WriteLine(num + " is an Armstrong Number");
        else
            Console.WriteLine(num + " is not an Armstrong Number");
    }
}

File: C/armstrong.c
#include <stdio.h>

int main() {
    int num, sum = 0, digit, temp;

    printf("Enter a number: ");
    scanf("%d", &num);

    temp = num;

    while (temp != 0) {
        digit = temp % 10;
        sum += digit * digit * digit;
        temp /= 10;
    }

    if (sum == num)
        printf("%d is an Armstrong Number\n", num);
    else
        printf("%d is not an Armstrong Number\n", num);

    return 0;
}